src/games/unpublishedGame.js

```diff
@@ -0,0 +1,51 @@
+import { nanoid } from "nanoid";
+
+const genGame = () => {
+  const items = [...Array(2000)].map((e, index) => ({
+    type: "rect",
+    x: 10 + index,
+    y: 10 + index,
+    width: 100,
+    height: 100,
+    id: nanoid(),
+  }));
+  return {
+    items,
+    availableItems: [
+      {
+        groupId: "Group",
+        label: "Rect",
+        type: "rect",
+        color: "#00D022",
+        width: 80,
+        height: 80,
+      },
+    ],
+    board: {
+      size: 4000,
+      scale: 0.5,
+      name: "Unpublished Game",
+      published: false,
+      translations: [
+        {
+          language: "fr",
```

Comment:
Is fr variant necessary?

Reply:
Not in this case. I wanted to keep the set of initial games consistent.